packages/backend/src/app.ts

```diff
@@ -61,6 +64,7 @@ const start = async () => {
       host: "0.0.0.0",
       port: 3000,
     });
+    await databaseConnection.runMigrations();
```

Comment:
I think migrations should run before starting the application, right? What happens if a query is performed while migrations are running?